lib/src/source/hosted.dart

```diff
@@ -62,10 +62,10 @@ class HostedSource extends Source {
     var url = io.Platform.environment['PUB_HOSTED_URL'];
     if (url == null) return null;
     var uri = Uri.parse(url);
-    if (uri.scheme?.isEmpty ?? true) {
+    if (uri.scheme != 'http' && uri.scheme != 'https') {
```

Comment:
`uri.hasScheme` is slightly more...uh...right

Handles weird casing, etc.

Reply:
Hmm - I think hasScheme only tells us if we have a scheme or not.
We want to ensure that the scheme is one of 'http' and 'https'.

But the casing part is a good point. Fixed!

Reply:
Ah you mean `uri.isScheme` didn't know about that!

Fixed!